python/sglang/srt/managers/scheduler.py

```diff
@@ -835,6 +848,10 @@ def event_loop_overlap(self):
             batch = self.get_next_batch_to_run()
             self.cur_batch = batch
 
+            if batch:
+                for req in batch.reqs:
+                    trace_event("schedule", req.rid)
```

Comment:
wrap this into a function called `trace_event_batch`.

The code in `event_loop_XXXX` should be very concise, only expose core logics